pycam/__init__.py

```diff
@@ -18,7 +18,8 @@
 along with PyCAM.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-VERSION = "0.6.1-dev"
+from pycam.version import VERSION  # noqa F401
```

Comment:
I prefer the `try: import Version; else VERSION="default"` idiom, since it doesn't require us to overwrite a git-tracked Version.py.  dbbbe6211bab0